cpp/src/randomforest/randomforest.hpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2018-2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <cuML.hpp>
+#include "randomforest.h"
+#include "decisiontree/decisiontree.h"
+
+namespace ML{
+
+struct RF_metrics {
+	float accuracy;
+
+	RF_metrics(float cfg_accuracy);
+	void print();
+};
+
+template <class T>
```

Comment:
@Salonijain27  Why do we have duplicate definition for rfClassifier class in .hpp as well as .h files. Is that way to just do it once.